### DISCUSSION TITLE
Does it support also chat GPT 3.5?

### DISCUSSION THREAD

**Discussion by BenK93** [freeedcom/ai-codereviewer#22] [Q&A]
If I created a API token in OpenAI but as a free user, will the CI work as well?

**Comment by villesau** [comment]
You can set the used model in the config:  https://github.com/freeedcom/ai-codereviewer#setup

` OPENAI_API_MODEL: "gpt-3.5-turbo"`

Not sure about free tier, please tell if you find out! :)
